src/lib.rs

```diff
@@ -459,14 +459,16 @@ pub struct QuicStatisticsRecv {
     /// Includes DuplicatePackets.
     pub dropped_packets: u64,
     pub duplicate_packets: u64,
-    /// Sum of UDP payloads
-    pub total_bytes: u64,
-    /// Sum of stream payloads
-    pub total_stream_bytes: u64,
```

Comment:
Why did you move these? From msquic.h:
```c
    struct {
        uint64_t TotalPackets;          // QUIC packets; could be coalesced into fewer UDP datagrams.
        uint64_t ReorderedPackets;      // Packets where packet number is less than highest seen.
        uint64_t DroppedPackets;        // Includes DuplicatePackets.
        uint64_t DuplicatePackets;
        uint64_t TotalBytes;            // Sum of UDP payloads
        uint64_t TotalStreamBytes;      // Sum of stream payloads
        uint64_t DecryptionFailures;    // Count of packet decryption failures.
        uint64_t ValidAckFrames;        // Count of receive ACK frames.
    } Recv;
```

Reply:
Oh, looks like I got the structs mixed up